src/webgl/p5.Camera.js

```diff
@@ -1390,7 +1390,8 @@ p5.Camera = class Camera {
     this.cameraMatrix.translate([tx, ty, tz]);
 
     if (this._isActive()) {
-      this._renderer.uMVMatrix.set(this.cameraMatrix);
+      this._renderer.uModelMatrix.set(this.modelMatrix);
```

Comment:
`this` is a p5.Camera here, which doesn't have a `modelMatrix` or `viewMatrix`, just a camera matrix. The errors this is logging are I think the cause of one of the test failures. I think this line and the one below should be replaced by:
```js
this._renderer.uModelMatrix.reset();
this._renderer.uViewMatrix.set(this.cameraMatrix);
```